index.html

```diff
@@ -13,44 +13,195 @@
   </head>
   <body>
     <div class="container">
-      <header>
-        <div class="logo-placeholder">
-          <a href="#" class="logo">My Logo</a>
+      <section class="home">
+        <header>
+          <div class="logo-placeholder">
+            <a href="#" class="logo">My Logo</a>
+          </div>
+          <nav class="nav-links">
+            <div class="menu-btn">
+              <img src="./images/menu.png" alt="Menu button" />
+            </div>
+          </nav>
+        </header>
+        <div class="main">
+          <h1 class="title">I'm Derrick Glad to see you!</h1>
+          <p class="about-me">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <h4 class="social-title">LET'S CONNECT</h4>
+          <ul class="social-icons">
+            <li>
+              <a href="#"><img src="./images/twitter.svg" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/Linkedi.svg" alt="Linkedln" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/gmail.svg" alt="Gmail" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/github.svg" alt="Github" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/fingers.svg" alt="Social media"
+              /></a>
+            </li>
+          </ul>
         </div>
-        <nav class="nav-links">
-          <div class="menu-btn">
-            <img src="./images/menu.png" alt="Menu button" />
+      </section>
+
+      <section class="works">
+        <div class="card">
+          <div class="card-bg card1"></div>
+          <div class="card-projects">
+            <h1 class="card-title">Tonic</h1>
+            <ul class="card-job-desc">
+              <li class="card-company">CANOPY</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-job">Back End Dev</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-year">2015</li>
+            </ul>
+            <p class="card-persona">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="card-languages">
+              <li class="card-lang1">html</li>
+              <li class="card-lang2">css</li>
+              <li class="card-lang3">javascript</li>
+            </ul>
+            <button type="button" class="card-submit-button">
+              See Project
+            </button>
```

Comment:
- [ ] [OPTIONAL] Great job styling your buttons so far :clap: . There is just one small thing left. Kindly add button interactions eg on hover, on click. This way it will be more intuitive for your users rather than the buttons just being static. If you might not be sure how to implement this, have a look at [this wonderful resource](https://zellwk.com/blog/style-hover-focus-active-states/).

**Below are the button states as provided in the chosen Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185753897-ba9f3a25-d358-4056-8755-d47a96bffc51.png)

Reply:
Thanks Boniface!